dotCMS/src/main/java/com/dotcms/publisher/bundle/business/BundleAPIImpl.java

```diff
@@ -391,4 +395,44 @@ public void deleteAssetFromBundle(String assetId, String bundleId)
 
 	}
 
+	/**
+	 * This takes a Bundle, generates the folder/file structure and returns the resulting directory
+	 * as a File handle. It will not delete the bundle directory if it already existed.
+	 * @param bundle - Bundle to generate
+	 * @return
+	 */
+    @CloseDBIfOpened
+    private File generateBundleDirectory(final Bundle bundle) {
+
+        final PushPublisherConfig pushPublisherConfig = new PushPublisherConfig(bundle);
+        pushPublisherConfig.setPublishers(Arrays.asList(GenerateBundlePublisher.class));
+        try {
+            APILocator.getPublisherAPI().publish(pushPublisherConfig);
+        }
+        catch(final Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6239861)